.mega-linter.yml

```diff
@@ -5,3 +5,11 @@ DISABLE_LINTERS:
   - SPELL_CSPELL
   - GROOVY_NPM_GROOVY_LINT
   - REPOSITORY_TRIVY
+  - PYTHON_BANDIT
```

Comment:
info: we have python code used as scripts. We don't wont to make it clean ;) Makeing those short scripts prettier, won't give much value.